app/models/mail_server_log.rb

```diff
@@ -256,7 +256,7 @@ def delivery_status
     DeliveryStatusSerializer.load(read_attribute(:delivery_status))
   end
 
-  private
+  private_class_method
```

Comment:
Changing this is now causing `Lint/UselessAccessModifier` warning so I think the wrong change has been made here.

Looking at the warning:
```
app/models/mail_server_log.rb:288:3: W: Lint/IneffectiveAccessModifier: private (on line 259) does not make singleton methods private. Use private_class_method or private inside a class << self block instead.
  def self.create_mail_server_logs(emails, line, order, done)
  ^^^
```

It appears this is because the `self.create_mail_server_logs` is under the `private` keyword. This is because it is a class method (EG it starts with `self.`).

We should retain the `private` keyword and resolve the this warning in a different way. This [resource shows how](https://medium.com/rubycademy/3-ways-to-make-class-methods-private-in-ruby-64b970e54613) this can be done.